src/Compilers/CSharp/Portable/FlowAnalysis/CSharpDataFlowAnalysis.cs

```diff
@@ -118,7 +118,8 @@ public override ImmutableArray<ISymbol> DefinitelyAssignedOnExit
 
         private (ImmutableArray<ISymbol> onEntry, ImmutableArray<ISymbol> onExit) ComputeDefinitelyAssignedValues()
         {
-            if (_definitelyAssignedOnEntry.IsDefault)
+            if (_definitelyAssignedOnEntry.IsDefault ||
```

Comment:
I think this is overkill. You can probably just check `_definitelyAssignedOnExit` and that should be sufficient.

Reply:
oh, duh.  true :)